packages/back-end/src/controllers/experiments.ts

```diff
@@ -96,14 +96,15 @@ export async function getExperimentsFrequencyMonth(
   const { num } = req.params;
   const experiments = await getExperimentsByOrganization(org.id, project);
 
-  const allData: { name: string; numExp: number }[] = [];
+  const allData: { date: string; numExp: number }[] = [];
 
   // make the data array with all the months needed and 0 experiments.
   for (let i = parseInt(num) - 1; i >= 0; i--) {
     const d = new Date();
+    d.setDate(1); // necessary because altering the month may result in an invalid date (ex: Feb 31)
     d.setMonth(d.getMonth() - i);
     const ob = {
-      name: format(d, "MMM yyy"),
+      date: format(d, "MM/dd/yyyy"),
```

Comment:
This date format is ambiguous, so when we are parsing dates later (especially on the front end when people have different locales), it might cause issues. For example, `01/02/2022` could be parsed as either Jan 2nd or Feb 1st. 

`yyyy-mm-dd` is much safer.